api/illumio/cloud/k8sclustersync/v1/cilium_flow.go

```diff
@@ -0,0 +1,68 @@
+package k8sclustersyncv1
+
+import (
+	"time"
+)
+
+var _ Flow = &CiliumFlow{}
+
+type CiliumFlowKey struct {
+	SourceIP           string
+	DestinationIP      string
+	SourcePort         int
+	DestinationPort    int
+	Protocol           string
+	SourceK8sMeta      string
+	DestinationK8sMeta string
+}
+
+func (flow *CiliumFlow) StartTimestamp() time.Time {
+	return flow.GetTime().AsTime()
+}
+
+func (flow *CiliumFlow) Key() any {
+	if flow == nil {
+		return nil
+	}
+
+	key := CiliumFlowKey{
+		SourceIP:           flow.Layer3.GetSource(),
+		DestinationIP:      flow.Layer3.GetDestination(),
+		SourceK8sMeta:      flow.SourceEndpoint.GetPodName(),
+		DestinationK8sMeta: flow.DestinationEndpoint.GetPodName(),
```

Comment:
The UID would have been a better choice. Because the name is not unique: multiple Pods can have the same name in different namespaces at the same time, and when a Pod in a StatefulSet restarts it gets the same name. But in all cases they all get different UIDs, so using UID is better.